channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpNativeBotRequest.kt

```diff
@@ -111,6 +119,26 @@ interface TelephonyBotRequest : JaicpNativeBotRequest {
     }
 }
 
+/**
+ * Contains the status of the transfer of a customer to an operator.
+ *
+ * @property status agent transfer status. It returns a string: `SUCCESS` or `FAIL`.
+ *
+ * @property hangup true means the customer has ended the conversation.
+ * false means the customer is still online and was re-directed to the bot (if continueCall: true in [TelephonySwitchReply]).
+ *
+ * @property number the phone number the call was transferred to.
+ */
+
+@Serializable
+data class TransferStatus(
+    val status: String,
```

Comment:
Is it possible to deserialize status into enum?

Reply:
Yes, I've made changes.